source/backend/dal/Repositories/DocumentActivityTemplateRepository.cs

```diff
@@ -0,0 +1,106 @@
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Security.Claims;
+using Microsoft.EntityFrameworkCore;
+using Microsoft.Extensions.Logging;
+using Pims.Core.Extensions;
+using Pims.Dal.Entities;
+
+namespace Pims.Dal.Repositories
+{
+    /// <summary>
+    /// DocumentActivityTemplateRepository class, provides a service layer to interact with activity template documents within the datasource.
+    /// </summary>
+    public class DocumentActivityTemplateRepository : BaseRepository<PimsActivityTemplateDocument>, IDocumentActivityTemplateRepository
+    {
+        #region Constructors
+
+        /// <summary>
+        /// Creates a new instance of a DocumentActivityTemplateRepository, and initializes it with the specified arguments.
+        /// </summary>
+        /// <param name="dbContext"></param>
+        /// <param name="user"></param>
+        /// <param name="logger"></param>
+        public DocumentActivityTemplateRepository(PimsContext dbContext, ClaimsPrincipal user, ILogger<DocumentActivityRepository> logger)
+            : base(dbContext, user, logger)
+        {
+        }
+        #endregion
+
+        #region Methods
+
+        /// <summary>
+        /// Get a list of all the document relationships for a given document.
+        /// </summary>
+        /// <returns></returns>
+        public IList<PimsActivityTemplateDocument> GetAllByDocument(long documentId)
+        {
+            return this.Context.PimsActivityTemplateDocuments
+                .Include(ad => ad.Document)
+                    .ThenInclude(d => d.DocumentStatusTypeCodeNavigation)
+                .Include(ad => ad.Document)
+                    .ThenInclude(d => d.DocumentType)
+                .Where(ad => ad.DocumentId == documentId)
+                .AsNoTracking()
+                .ToList();
+        }
+
+        /// <summary>
+        /// Get a list of all the document relationships for a given activity template.
```

Comment:
nit: might be all the comments in this repo still refer to document relationships when this repo is specific to templates.

Reply:
this one is pretty close, at the end of the doc there is the specification that this is for an activity template